### DISCUSSION TITLE
Welcome to BlueBuild!

### DISCUSSION THREAD

**Discussion by xynydev** [blue-build/.github#10] [Announcements]
Universal Blue's old custom image tooling has now been fully migrated over to BlueBuild.

Read the full announcement and introduction post: https://blue-build.org/blog/introducing-bluebuild/

![](https://github.com/blue-build/.github/assets/60004820/337323ed-70e4-4025-8c73-e8fe0c183c7c)

**Comment by flexagoon** [comment]
I just migrated my image to BlueBuild and detached the repo from the uBlue template. However, since the repo is no longer attached to a template, how will I be able to sync my config with the changes in the upstream template?

**Reply by xynydev** [reply]
This was asked and answered on Discord already, but I'll answer here again.

Most of the logic is in the GitHub Action and CLI, which is automatically updated by GitHub's Dependabot. If there is a desire to sync with the other parts of the template, though, a guide is planned and will be written as soon as I have time (or someone else contributes it).
